scrapy/core/scheduler.py

```diff
@@ -73,6 +97,7 @@ def next_request(self):
                 self.stats.inc_value('scheduler/dequeued/disk', spider=self.spider)
         if request:
             self.stats.inc_value('scheduler/dequeued', spider=self.spider)
+            self.stats.inc_value('scheduler/dequeued/key/{}'.format(self.request_key(request)))
```

Comment:
if `enqueued` stat is removed, this one should be removed as well

Reply:
Good point.